.travis.yml

```diff
@@ -121,6 +121,7 @@ before_install:
     - deactivate
     - virtualenv --system-site-packages ~/venv
     - source ~/venv/bin/activate
+    - sudo apt-get install libboost-filesystem1.55-dev libboost-python1.55-dev libboost-system1.55-dev libboost-thread1.55-dev
```

Comment:
Why don't these lines do the trick?
https://github.com/NVIDIA/DIGITS/blob/776131dc746f3bd09b685ae32c6d4c4b11c03394/.travis.yml#L86-L89